src/Input/TextField/TextFieldStyle.ts

```diff
@@ -9,13 +9,25 @@ export const IconContainer = styled.div`
 
   svg {
     fill: ${Greyscale.grey};
+    width: 10px;
+    height: 14px;
```

Comment:
I afraid that this change makes password icons too small.

![Screen Shot 2020-11-12 at 4 02 07 PM](https://user-images.githubusercontent.com/13249748/98912090-98e05b00-2500-11eb-9e21-59aad6ada5fe.png)

Reply:
Oh this change has to be in StartIconContainer not the IconContainer -_-

Reply:
This change is not required actually. :P